releases/release-1.31/release-notes/maps/pr-125359-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 125359
+releasenote:
+  text: 'Fix: during the kube-controller-manager restart, when the corresponding Endpoints
```

I don't think this was resolved? 
```suggestion
  text: 'Fixed: during the kube-controller-manager restart, when the corresponding Endpoints
```